CorsixTH/Lua/map.lua

```diff
@@ -279,6 +264,46 @@ function Map:load(level, difficulty, level_name, map_file, level_intro, map_edit
   return objects
 end
 
+-- A set of the TH campaign levels with additional CorsixTH config files, found in CorsixTH/Levels.
+local additional_config = list_to_set({"05", "07", 11, 12})
+
+--! Load further level configurations for the main campaign.
+--!param difficulty (string)
+--!param level_number (integer)
+--!param config (table) The level config created so far
+--!return error (string) Localised error message if error happens
+--!return config (table) Level config, if loaded successfully
+function Map:_loadCampaignLevel(difficulty, level_number, config)
```

Comment:
Could we make this function name even more obvious it's not for custom campaigns?
`_loadOriginalCampaignLevel`?

Reply:
Agreed, done.